move/axelar_gateway/sources/types/message_ticket.move

```diff
@@ -2,11 +2,12 @@ module axelar_gateway::message_ticket;
 
 use std::ascii::String;
 
-
-
 // -----
 // Types
 // -----
+/// This hot potato object is created to capture all the information about a remote contract call.
+/// In can then be submitted to the gateway to send the Message.
+/// It is advised that modules return this Message ticket to be submitted by the frontend, to ensure futureproofing the modules (submission might change).
```

Comment:
```suggestion
/// It is advised that modules return this Message ticket to be submitted by the frontend, so that when the gateway package is upgraded, the app doesn't need to upgrade as well, ensuring forward compatibility.
```